src/Proto/Semantic_JSON.hs

```diff
@@ -0,0 +1,842 @@
+-- Code generated by protoc-gen-jsonpb_haskell 0.1.0, DO NOT EDIT.
```

Comment:
`proto-lens` doesn't currently support encoding/decode to json so this is a bit of a hack to preserve semantic's existing API and functionality. This file gets generated by https://github.com/tclem/proto-lens-jsonpb, a protoc plugin and haskell library for encoding/decoding jsonpb which I ported over from proto3-suite. See https://developers.google.com/protocol-buffers/docs/proto3#json for how protobufs are suppose to translate to json.